src/render/boolean.ts

```diff
@@ -1,6 +1,15 @@
-export function isTruthy (val: any): boolean {
-  return !isFalsy(val)
+import { Context } from '../context/context'
+
+export function isTruthy (val: any, ctx: Context): boolean {
+  if ((val === null) && ctx && ctx.opts && ctx.opts.jsTruthy ) return false;
```

Comment:
`ctx.opts.jsTruthy` is guaranteed to exist so there's no need to `ctx && ctx.opts`...